resources/windows/wix/registryEntries.wxs.erb

```diff
@@ -47,6 +47,13 @@
             Type="string"
             Value="[PUPPET_AGENT_STARTUP_MODE]" />
         </RegistryKey>
+      </Component>
+
+      <Component Id="EventLogRegistryEntries"
+        Guid="3A4E2A16-1919-4BAF-A4CF-9FA3543BB27C"
+        Directory="TARGETDIR"
+        Win64="no"
+        permanent="yes">
```

Comment:
This was necessary, simply to add `permanent="yes"` to the relevant registry entries.